internal/api/github/client.go

```diff
@@ -48,13 +48,14 @@ func (c *Client) GetConfiguration(ctx context.Context, ownerLogin, repoName stri
 	// Try to download the contents of the configuration file.
 	ctxTimeout, fn := context.WithTimeout(ctx, DefaultGitHubOperationTimeout)
 	defer fn()
-	r, err := c.githubClient.Repositories.DownloadContents(ctxTimeout, ownerLogin, repoName, configuration.ConfigurationFilePath, nil)
+	r, res, err := c.githubClient.Repositories.DownloadContents(ctxTimeout, ownerLogin, repoName, configuration.ConfigurationFilePath, nil)
 	if err != nil {
-		if strings.Contains(err.Error(), "No file named") { // No better way of distinguishing between errors.
+		if strings.Contains(strings.ToLower(err.Error()), "no file named") { // fixme: look at status code instead of body
```

Comment:
Shall we submit an issue for that `fixme` or is this still WIP?

Reply:
Good idea, I'll create an issue for this.

Reply:
https://github.com/form3tech-oss/github-team-approver/issues/33